src/plasma-cash/db.ts

```diff
@@ -22,6 +21,8 @@ export class PlasmaDB {
     let adapter
     if (typeof localStorage === 'undefined' || localStorage === null) {
       const FileSync = require('lowdb/adapters/FileSync')
+      const shelljs = require('shelljs')
```

Comment:
This should be necessary because `shelljs` is marked as external and webpack should by and large ignore the import (https://github.com/loomnetwork/loom-js/blob/master/webpack.config.js#L24)